src/components/input/Input.tsx

```diff
@@ -1,10 +1,21 @@
 import { ComponentProps } from "react";
-import styles from "./index.module.css";
+import styles from "./Index.module.css";
+import classNames from "classnames";
 
 interface IInputProps extends Omit<ComponentProps<"input">, "type"> {
   type?: "text" | "number" | "email" | "password" | "url";
+  variant?: "normal" | "error" | "";
```

Comment:
No need for this variant. It would be better to add a prop called 'error'.

`error?: boolean;`